noobaa_sa/bucket.py

```diff
@@ -0,0 +1,121 @@
+"""
+Module which contain bucket operations like create, delete, list and update
+"""
+
+import logging
+import re
+
+from framework import config
+from framework.connection import SSHConnection
+from noobaa_sa.defaults import MANAGE_NSFS
+import noobaa_sa.exceptions as e
+
+log = logging.getLogger(__name__)
+
+
+
```

Comment:
Please use the existing "snake case" naming convention (lowercase_separated_by_underscores) instead of camel case for the methods of this class.

Reply:
Fixed